specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json

```diff
@@ -0,0 +1,95 @@
+{
+    "parameters": {
+        "Ocp-Apim-Subscription-Key": "{API key}",
+        "body": {

```

Comment:
>body [](start = 9, length = 4)

According to your swagger this parameter should be named as `input` and not body, right?

Reply:
No. `input` and `body` are not part of the request body. I think this is just a reference ID for the example body which has a `documents` root object. I have followed the same pattern as existing endpoints (the input for all is the same).

Reply:
Here's an example request body:

<code>
{
    "documents": [
                {
                    "language": "en",
                    "id": "1",
                    "text": "Microsoft released Windows 10"
                },
                {
                    "language": "en",
                    "id": "2",
                    "text": "In 1975, Bill Gates III and Paul Allen founded the company."
                }
            ]
}
</code>

Reply:
and I think all your samples are flagged as wrong
```
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Key Phrases".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Key Phrases", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Detect Language".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Detect Language".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Detect Language", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Sentiment".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Sentiment".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Sentiment", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Entities request" in operation "Entities".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Entities request" in operation "Entities".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Entities", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
```

Reply:
Thanks. Should be fixed now.